### ISSUE TITLE
sp - use publicUrl instead of public.host

### ISSUE THREAD

**Issue by severo** [georchestra/georchestra#2352]
Th security proxy is the only module that does not use the `publicUrl` property (it uses `public.host` instead). It might be the source of errors.
See also the https://github.com/georchestra/datadir repository.

**Comment by fvanderbiest** [comment]
Yes, `public.host=${publicUrl}` in https://github.com/georchestra/datadir/blob/f4377ae1f27813e0e522b632c4c8147af80ea670/security-proxy/security-proxy.properties#L2

I'm not sure we can get rid of it.
What's your proposal here ?

**Comment by severo** [comment]
Yes, in master it does no more refer to publicUrl: https://github.com/georchestra/datadir/blob/master/security-proxy/security-proxy.properties

My proposal is to modify the code in security proxy to refer to publicUrl, instead of public.host, nothing more.

**Comment by fvanderbiest** [comment]
Oh yes, forgot that ;-)

**Comment by severo** [comment]
Already fixed with #2384 and https://github.com/georchestra/datadir/pull/109. We can close.
